crates/tests/tests-common/src/common_tests/configuration_v3_tests.rs

```diff
@@ -75,11 +75,7 @@ async fn read_configuration(
     let contents =
         fs::read_to_string(absolute_configuration_directory.join("configuration.json")).await?;
 
-    let mut multi_version: serde_json::Value = serde_json::from_str(&contents)?;
-
-    // We assume the stored NDC metadata file to be in the newest version, so to be able to make
```

Comment:
The check for version tags have become the responsibility of each version implementation. Therefore we shouldn't remove it.